iroha-android-sample/src/main/java/jp/co/soramitsu/iroha/android/sample/interactor/CreateAccountInteractor.java

```diff
@@ -0,0 +1,94 @@
+package jp.co.soramitsu.iroha.android.sample.interactor;
+
+
+import com.google.protobuf.InvalidProtocolBufferException;
+
+import java.math.BigInteger;
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import io.grpc.ManagedChannel;
+import io.reactivex.Completable;
+import io.reactivex.Scheduler;
+import iroha.protocol.BlockOuterClass;
+import iroha.protocol.CommandServiceGrpc;
+import jp.co.soramitsu.iroha.android.ByteVector;
+import jp.co.soramitsu.iroha.android.Keypair;
+import jp.co.soramitsu.iroha.android.ModelCrypto;
+import jp.co.soramitsu.iroha.android.ModelProtoTransaction;
+import jp.co.soramitsu.iroha.android.ModelTransactionBuilder;
+import jp.co.soramitsu.iroha.android.UnsignedTx;
+import jp.co.soramitsu.iroha.android.sample.PreferencesUtil;
+import jp.co.soramitsu.iroha.android.sample.injection.ApplicationModule;
+
+import static jp.co.soramitsu.iroha.android.sample.Constants.CONNECTION_TIMEOUT_SECONDS;
+import static jp.co.soramitsu.iroha.android.sample.Constants.CREATOR;
+import static jp.co.soramitsu.iroha.android.sample.Constants.DOMAIN_ID;
+import static jp.co.soramitsu.iroha.android.sample.Constants.PRIV_KEY;
+import static jp.co.soramitsu.iroha.android.sample.Constants.PUB_KEY;
+import static jp.co.soramitsu.iroha.android.sample.Constants.TX_COUNTER;
+
+public class CreateAccountInteractor extends CompletableInteractor<String> {
+
+    private final ManagedChannel channel;
+    @Inject
+    ModelCrypto crypto;
```

Comment:
Why do you inject model cpypto? Does it needed?

Reply:
We need it to generate keys on 55 line